components/org.wso2.openbanking.cds.demo.backend/src/main/webapp/WEB-INF/web.xml

```diff
@@ -39,12 +39,143 @@
+            </param-value>
+        </init-param>
+
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+
+    <servlet>
+        <servlet-name>CommonServiceJAXServlet</servlet-name>
+        <display-name>CommonService JAX-WS/JAX-RS Servlet</display-name>
+        <description>CommonService JAX-WS/JAX-RS Endpoint</description>
+        <servlet-class>
+            org.apache.cxf.jaxrs.servlet.CXFNonSpringJaxrsServlet
+        </servlet-class>
+        <init-param>
+            <param-name>service-list-stylesheet</param-name>
+            <param-value>servicelist.css</param-value>
+        </init-param>
+        <init-param>
+            <param-name>jersey.config.server.provider.classnames</param-name>
+            <param-value>org.glassfish.jersey.media.multipart.MultiPartFeature</param-value>
+        </init-param>
+
+        <init-param>
+            <param-name>jaxrs.serviceClasses</param-name>
+            <param-value>
+                org.wso2.openbanking.cds.demo.backend.services.CommonService
+            </param-value>
+        </init-param>
+
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+
+    <servlet>
+        <servlet-name>DiscoveryServiceJAXServlet</servlet-name>
+        <display-name>DiscoveryService JAX-WS/JAX-RS Servlet</display-name>
+        <description>DiscoveryService JAX-WS/JAX-RS Endpoint</description>
+        <servlet-class>
+            org.apache.cxf.jaxrs.servlet.CXFNonSpringJaxrsServlet
+        </servlet-class>
+        <init-param>
+            <param-name>service-list-stylesheet</param-name>
+            <param-value>servicelist.css</param-value>
+        </init-param>
+        <init-param>
+            <param-name>jersey.config.server.provider.classnames</param-name>
+            <param-value>org.glassfish.jersey.media.multipart.MultiPartFeature</param-value>
+        </init-param>
+
+        <init-param>
+            <param-name>jaxrs.serviceClasses</param-name>
+            <param-value>
+                org.wso2.openbanking.cds.demo.backend.services.DiscoveryService
+            </param-value>
+        </init-param>
+
         <load-on-startup>1</load-on-startup>
     </servlet>
 
+    <servlet>
+        <servlet-name>CustomerDetailsJAXServlet</servlet-name>
+        <display-name>CustomerDetails JAX-WS/JAX-RS Servlet</display-name>
+        <description>CustomerDetails JAX-WS/JAX-RS Endpoint</description>
+        <servlet-class>
+            org.apache.cxf.jaxrs.servlet.CXFNonSpringJaxrsServlet
+        </servlet-class>
+        <init-param>
+            <param-name>service-list-stylesheet</param-name>
+            <param-value>servicelist.css</param-value>
+        </init-param>
+        <init-param>
+            <param-name>jersey.config.server.provider.classnames</param-name>
+            <param-value>org.glassfish.jersey.media.multipart.MultiPartFeature</param-value>
+        </init-param>
+
+        <init-param>
+            <param-name>jaxrs.serviceClasses</param-name>
```

Comment:
isnt this "jaxrs.address" required to define the address?

Reply:
It is needed if we are defining the address inside a `<servlet>`. Here we are using multiple servlets with a servlet mapping.